dpctl/_sycl_context.pyx

```diff
@@ -485,3 +485,12 @@ cdef api DPCTLSyclContextRef get_context_ref(SyclContext ctx):
     :class:`dpctl.SyclContext` instance.
     """
     return ctx.get_context_ref()
+
+
+cdef api SyclContext make_SyclContext(DPCTLSyclContextRef CRef):
```

Comment:
Why are these changes here?

Reply:
Because without them pybind11 casters can not be written. One needs 2 way street to go from Python object to c++ class, and back.